### PR TITLE
DR-2295 Make logs human-readable locally

### PR DESCRIPTION
**Pull request by tlangs**
Disabling trace logging, and also making local logs human-readable again. Due to a change in Terra Commons Lib, the `TDR_LOG_APPENDER` env var is no longer respected, so we need to activate the `human-readable-logging` spring profile to get non-JSON logging locally.

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -28,8 +28,10 @@ terra.common.stairway.forceCleanStart=false
 terra.common.stairway.migrateUpgrade=true
 terra.common.stairway.maxParallelFlights=120
 terra.common.stairway.clusterNameSuffix=stairwaycluster
+terra.common.stairway.tracing-enabled=false
```
Comment (pshapiro4broad) [inline src/main/resources/application.properties:31]: Where is this property used? I looked in `terra-common-library` and `stairway` but didn't see it mentioned in either place. 

Actually I don't see many of these stairway properties anywhere, so I must be looking in the wrong place. Do you know where they're used?

**Review by pshapiro4broad — APPROVED** [review]
Looks OK, as far as I can tell. If `TDR_LOG_APPENDER` is no longer supported / meaningful, then it should be removed from the source code as well. It is currently in our GitHub workflows, the readme, and the logback config. Also, should we delete our logback config completely if tcl's config overrides ours?

**Review by snf2ye — APPROVED** [review]
I agree w/ @pshapiro4broad , it would be great to ditch references to TDR_LOG_APPENDER if it's not used anymore. But, otherwise LGTM - thanks for figuring this out!

File: src/main/resources/application.properties (hunk 1 above)
Comment (tlangs) [reply src/main/resources/application.properties:31]: I got the properties from here: https://github.com/DataBiosphere/terra-resource-buffer/blob/master/src/test/resources/application-test.yml

**Comment by tlangs** [comment]
We still need `TDR_LOG_APPENDER` there to make it do the stackdriver logging format, but we also need `human-readable-logging` locally now. I tried all combinations.

**Review by nmalfroy — COMMENTED** [review]
Will human readable be on in our cloud deployments now? Or do profiles all get overridden?

**Review by samanehsan — APPROVED** [review]

**Pull request merged**